src/index.js

```diff
@@ -0,0 +1,103 @@
+var Post = require('./post/post.model');
+var express = require('express');
+var app = express();
+var posts = [];
+
+var addPost = function(post) {
```

Comment:
Why not use arrow function and use `const` over `var`, related to `getPostById`, `removePost`, `getPosts` as well.

```suggestion
const addPost = (post) => {
```